rust/src/conf.rs

```diff
@@ -113,9 +100,7 @@ impl ConfNode {
 
         unsafe {
             let s = CString::new(key).unwrap();
-            if ConfGetChildValueBool(self.conf,
-                                     s.as_ptr(),
-                                     &mut vptr) != 1 {
+            if ConfGetChildValueBool(self.conf, s.as_ptr(), &mut vptr) != 1 {
```

Comment:
I don't get why some things like the test stuff above are expanded into many line bloated things, while this is made more compact. I like the more compact stuff like this

Reply:
Struct literals and function arguments are treated differently.